google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -77,7 +77,10 @@ interfaces:
     long_running:
       return_type: google.cloud.speech.v1.LongRunningRecognizeResponse
       metadata_type: google.cloud.speech.v1.LongRunningRecognizeMetadata
-      polling_interval_millis: 20000
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```

Comment:
I believe in the last discussion the 'max_poll_delay_millis' should be at least 60 seconds for Speech. I will double check with the team